microk8s-resources/actions/common/utils.sh

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
+exit_if_no_sudo() {
```

Comment:
Strictly speaking this name is a bit misleading - might be better to call it `exit_if_no_permission` perhaps?

Reply:
Done